core/src/main/java/io/questdb/cairo/RecordSinkFactory.java

```diff
@@ -69,6 +69,9 @@ public static RecordSink getInstance(
         final int rGetSym = asm.poolInterfaceMethod(Record.class, "getSym", "(I)Ljava/lang/CharSequence;");
         final int rGetBin = asm.poolInterfaceMethod(Record.class, "getBin", "(I)Lio/questdb/std/BinarySequence;");
         final int rGetRecord = asm.poolInterfaceMethod(Record.class, "getRecord", "(I)Lio/questdb/cairo/sql/Record;");
+        final int rGetUuidMostSig = asm.poolInterfaceMethod(Record.class, "getUuidMostSig", "(I)J");
```

Comment:
can we make UUID interchangeable with LONG28 for simplicity?

Also, we should follow existing vocabulary, most/least -> Lo/Hi

Reply:
yeah, that's a good idea. little endian for everything.